.gitignore

```diff
@@ -2,6 +2,8 @@ generated/
 node_modules/
 components/*/test/screenshots/current/
 components/*/test/screenshots/golden/
+helpers/test/screenshots/current/
+helpers/test/screenshots/golden/
```

Comment:
I'd love to re-organize everything so that tests & demos have their own top-level directory instead of nested inside each component. Would make `eslint` config easier too.